docs/install.md

```diff
@@ -0,0 +1,84 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Hyperledger Private Data Objects, Sample Contracts #
```

Comment:
Nit: Would rename this to something like "PDO Sample Contracts Installation". Otherwise this is the same title as the top-level readme.

Reply:
Done